python/test/unit/language/test_core.py

```diff
@@ -1744,7 +1763,10 @@ def kernel(in_ptr, idx_ptr, out_ptr, shape0, shape1, mask_step, XBLOCK: tl.const
             cnt += 1
 
     kernel[(1, )](val, idx, dst, shape0, shape1, mask_step, 64, num_ctas=num_ctas)
-    np.testing.assert_allclose(to_numpy(dst_ref), to_numpy(dst), atol=1e-2)
+
+    # Do not check accuracy for bf16
```

Comment:
Why do you ignore the accuracy checks everywhere? What's the point of running the test at all if you don't check that it outputs the correct result.

Reply:
@peterbell10 One thing I will try is keeping an alternative set of checks but with lower accuracy and see if those pass on AMD and NV. If this works, I will land that.

Reply:
> have you tried trunking the low mantissa bits and comparing to f32? I would expect it to be close

Ah I like this and I just added this (looked at what the reduce tests were doing).
I did still have to alter the relative checks in a couple places.